framework/Models/HybridModel.py

```diff
@@ -0,0 +1,621 @@
+
+    return allValid
+
+  def _extractInputs(self,dataIn, paramsList):
+    """
+      Extract the the parameters in the paramsList from the given data object dataIn
+      @ dataIn, Instance or Dict, data object or dictionary contains the input and output parameters
+      @ paramsList, List, List of parameter names
+      @ localInput, dict,
+    """
+    localInput = dict.fromkeys(paramsList, None)
+    if type(dataIn) == dict:
+      for elem, value in dataIn.items():
+        if elem in paramsList:
+          localInput[elem] = np.atleast_1d(value)
+    else:
+      self.raiseAnError(IOError, "The input type '", inputType, "' can not be accepted!")
+
+    return localInput
+
+  def computeCDCoefficient(self, trainSet, newSet):
+    """
+      This function will compute the Crowding distance coefficients among the input parameters
+      @ In, trainSet, numpy.array, array contains values of previous generated input parameters
+      @ In, newSet, numpy.array, array contains values of current generated input parameters
+      @ Out, crowdingDistCoeff, float, the coefficient of crowding distance
+    """
+    totalSet = np.concatenate((trainSet,newSet), axis=1)
+    dim = totalSet.shape[1]
+    distMat = np.zeros((dim, dim))
+    for i in range(dim):
+      for j in range(dim):
+        distMat[i,j] = linalg.norm(totalSet[:,i] - totalSet[:,j])
+    crowdingDist = np.sum(distMat,axis=1)
+    maxDist = np.amax(crowdingDist)
+    minDist = np.amin(crowdingDist)
+    if maxDist == minDist:
+      crowdingDistCoeff = 1.0
+    else:
+      crowdingDistCoeff = (maxDist - crowdingDist[-1])/(maxDist - minDist)
+
+    return crowdingDistCoeff
+
+  def checkTrainingSize(self):
+    """
+      This function will check the size of existing training set
+      @ In, None
+      @ Out, newGeneratedTrainingSize, int, the size of existing training set
+    """
+    newGeneratedTrainingSize = len(self.tempTargetEvaluation) - self.existTrainSize
+    self.raiseADebug("New generated training size is: ", newGeneratedTrainingSize)
+    return newGeneratedTrainingSize
+
+  def submit(self,myInput,samplerType,jobHandler,**kwargs):
+    """
+      This will submit an individual sample to be evaluated by this model to a
+      specified jobHandler. Note, some parameters are needed by createNewInput
+      and thus descriptions are copied from there.
+      @ In, myInput, list, the inputs (list) to start from to generate the new one
+      @ In, samplerType, string, is the type of sampler that is calling to generate a new input
+      @ In,  jobHandler, JobHandler instance, the global job handler instance
+      @ In, **kwargs, dict,  is a dictionary that contains the information coming from the sampler,
+        a mandatory key is the sampledVars'that contains a dictionary {'name variable':value}
+      @ Out, None
+    """
+    self.submitAsClient(myInput, samplerType, jobHandler, **kwargs)
+
+  def submitAsClient(self,myInput,samplerType,jobHandler,**kwargs):
```

Comment:
submitAsClient is not needed. Should be just the ```submit``` method

Reply:
fixed.